content/id/docs/reference/glossary/persistent-volume-claim.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Persistent Volume Claim
+id: persistent-volume-claim
+date: 2018-04-12
+full_link: /docs/concepts/storage/persistent-volumes/
+short_description: >
+  Mengklaim sumber daya penyimpanan yang didefinisikan di dalam suatu PersistentVolume, sehingga PersistentVolume tersebut dapat dipasang (_mounted_) sebagai sebuah volume pada suatu Container.
```

Comment:
Better to keep “volume” as English word because “volume” in Indonesia word seems not match with context.
```suggestion
  Mengklaim sumber daya penyimpanan yang didefinisikan di dalam suatu PersistentVolume, sehingga PersistentVolume tersebut dapat dipasang (_mounted_) sebagai sebuah _volume_ pada suatu Container.
```